_rules/video-alternative-c5a4ea.md

```diff
@@ -96,8 +96,8 @@ A video element with a track element that contains descriptions.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/video.webm" type="video/webm">
 	<track
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`